index.html

```diff
@@ -22,61 +22,89 @@ <h1 id="title">Dr. Norman Borlaug</h1>
           hunger."
         </figcaption>
       </figure>
+
       <div id="tribute-info">
         <h2>Here's a time line of Dr. Borlaug's life:</h2>
         <ul>
+          <li>1914 - Born in Cresco, Iowa</li>
```

Comment:
Consider using span to enclose dates of all the biography lists and then style them to be bold.

You could also increase the font size so it stands out even more.